images/icons/check.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" ?><svg id="Layer_1" style="enable-background:new 0 0 128 128;" version="1.1" viewBox="0 0 128 128" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><style type="text/css">
```

Comment:
Where did this SVG come from? Do we need to provide attribution?

Reply:
No attribution strictly necessary under its license https://www.iconfinder.com/icons/1398912/circle_correct_mark_success_tick_yes_check_icon